### ISSUE TITLE
vg construct example

### ISSUE THREAD

**Issue by dd2019d** [vgteam/vg#2384]
hi,

this is not an issue. plz bear with me since i am new to this. in the example to construct, you use the ".vcf.gz" file. currently, i have the ".fna" file, but how or where do i get the corresponding ".vcf.gz" file?

thanks.

**Comment by ekg** [comment]
The vcf.gz file represents variation against a reference, usually in FASTA
format (often written as .fna).

What is in your .fna file?

**Comment by dd2019d** [comment]
First line says: 

>CM008287.1 Fusarium oxysporum f. sp. ridicis-cucumerinum strain Forc016 chromosome 1, whole genome shotgun sequence

This is a reference file named "GCA_001702695.2_ASM170269v2_genomic.fna". I thought I could use this file instead of ".fa" file shown in the example.

**Comment by ekg** [comment]
Yes, you should be able to. But without variation information or multiple
references there is not a benefit to using vg.

**Comment by dd2019d** [comment]
How or where do I obtain the .vcf file for my data?

**Comment by glennhickey** [comment]
@ekg has a good overview of the various input types here: https://github.com/ekg/alignment-and-variant-calling-tutorial

**Comment by maxineliu** [comment]
I have the same question as dd2019d. I only have the reference genome of my research organism in `.fa` file and long-reads sequences of several samples in `.bam` file. I wonder how to obtain the VCF file? How to construct my pangenome?

What I think now is that first I need tools like samtool to convert `.bam` to `.fasta`, then construct a flat reference genome with `vg construct` , then do alignment - augumentation - varients calling to generate calls.vcf, finally add these VCF files to reference genome. 

Am I right? (Actually I don't think it's right. Really need help.)

**Comment by glennhickey** [comment]
To make a pangenome from long reads you can either:
1) Create a VCF using a *variant caller*, then use `vg construct` or
2) Use an *assembler* make an assembly for each sample, then align the assemblies together with either `cactus` or `pggb`

For 1), you can in theory use `vg augment/call` as the variant caller, but it's not going to work well for long reads (especially if they are noisy). And since you'll be calling against a linear reference, there's not much point to using `vg` anyway, so you'd be best served to find another tool more suited to your reads, and use that tool's output VCF as input to vg.

**Comment by faithokamoto** [comment]
The `vg construct --vcf` option may be used multiple times so no external merge is needed.
